modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -8033,6 +8034,8 @@ issuer-url=Issuer URL
 issuer-url-help=Set the issuer URL for the OpenID Connect provider.
 issues-tracking=Issues Tracking
 it-accepts-only-numbers=It accepts only numbers.
+it-is-not-allowed-to-create-update-action-on-after-delete-trigger=It is not allowed to create Update Action on "On After Delete" trigger. Please change the Action trigger.
```

Comment:
This is a sentence, so the case should be:
`It is not allowed to create an update action on "On After Delete" trigger. Please change the action trigger.`
Also added "an" before "update action" because of the next line, which uses it.